frontend/src/components/transactions/ReceiptOutcomeRow.tsx

```diff
@@ -0,0 +1,260 @@
+import React from "react";
+import { Row, Col } from "react-bootstrap";
+
+import * as T from "../../libraries/explorer-wamp/transactions";
+import { truncateAccountId } from "../../libraries/formatting";
+import { displayArgs } from "./ActionMessage";
+
+export interface Props {
+  sirReceiptHash: string;
+  receiptsObj: {};
+  receiptsInfoObj: {};
+  senderIsReceiverActions: T.RpcAction[];
+  senderIsReceiver: Boolean;
+}
+
+export default class extends React.Component<Props> {
+  handleStatusInfo = function (receipt: T.ReceiptLocalOutcome) {
+    let statusInfo;
```

Comment:
I feel like you don't need this variable, whenever you assign a value to it in this function, you may just as well return it immediately